vmdb/app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -2,6 +2,8 @@
 
 module EmsRefresh::Parsers
   class Openstack < Cloud
+    include EmsRefresh::Parsers::OpenstackCommon::OrchestrationStacks
```

Comment:
hm actually openstack and openstack_infra will differ, so it made sense for me to extract it to common lib used by both, rather than using one of them

Also extracting it per entity it load will be nice, so we will have 
EmsRefresh::Parsers::OpenstackCommon::Flavors
EmsRefresh::Parsers::OpenstackCommon::Images
etc.

all in separate shared directory

OR you just mean to use EmsRefresh::Parsers::Openstack as namespace? I was thinking to just include EmsRefresh::Parsers::OpenstackCommon which will include all modules inside. We can't do that when namespace is a class.